api.ts

```diff
@@ -211,6 +211,9 @@ export interface SyncTableDef<
   entityName?: string;
   /** See {@link DynamicOptions.defaultAddDynamicColumns} */
   defaultAddDynamicColumns?: boolean;
+  /** See {@link SyncTableOptions.maxUpdateBatchSize} */
+  /** @hidden */
+  maxUpdateBatchSize?: number;
```

Comment:
If the goal is to put `executeUpdate` next to `execute`, that would be in `SyncFormula`/`SyncFormulaDef`. If the goal is to put this field next to `executeUpdate`, then it would go there too, not here on `SyncTableDef`.

Reply:
yeah I had the question as well in the first place. It does look like `maxUpdateBathSize` should be a property of the sync table but not to a particular formula so this change makes sense. 

and as it is to the `executeUpdate`, it becomes an interesting question as if makes sense for it to live in the `getter` in the longer term. and if we decided `getter` should be `formula` then it would make sense to have `maxUpdateBathSize` there as well.

Reply:
Yeah I think the getter is the right place to put this stuff, it's just named poorly at the moment. Not sure the feasibility of renaming though, but at least it's an internal field name even if the misnomer sticks around.

Reply:
In our discussion, we had decided to put this metadata next to other table fields, not within the formula.  Happy to change that if it makes sense, though.  Just tell me where I should put it.

```
 makeDynamicSyncTable({
    name: 'Tasks',
    identityName: 'Tasks',
    connectionRequirement: ConnectionRequirement.Required,
    listDynamicUrls: handlers.listDynamicTaskUrls,
    getName: handlers.getDynamicTaskName,
    getDisplayUrl: handlers.getDynamicTaskDisplayUrl,
    getSchema: handlers.getDynamicTaskSchema,
    maxUpdateBatchSize: 15, // defaults to 1, min 1, max 1000?
    ...
```

Reply:
Clarified over slack that we indeed want to move this into the formula.  Done.